### PR TITLE
New translations for translation/ru/narinyanievmenenko/sutta/kn/thig

### PR DESCRIPTION
**Pull request by sc-translatatron**
Request made by neptunao

Please do not modify this branch directly. Changes should be
made via the Bilara Translation App and the Pull Request
updated from Bilara.

### CODE REVIEW

File: translation/ru/narinyanievmenenko/sutta/kn/thig/thig2.7_translation-ru-narinyanievmenenko.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "thig2.7:0.1": "Стихи старших монахинь 2.7 ",
+  "thig2.7:0.2": "Две строфы ",
+  "thig2.7:0.3": "Митта Тхери ",
+  "thig2.7:1.1": "По лунным дням я соблюдала предписанья —",
+  "thig2.7:1.2": "На день восьмой, четырнадцатый, пятнадцатый; ",
+  "thig2.7:1.3": "Восемь предписаний держала, ",
+  "thig2.7:1.4": "Стремясь к рождениям в чертогах дэвов. ",
+  "thig2.7:2.3": "Теперь я пищу принимаю лишь раз в день, ",
+  "thig2.7:2.4": "С обритой головою, <j>Закутанная в монашеское одеянье. ",
```
Comment (ccronje) [inline translation/ru/narinyanievmenenko/sutta/kn/thig/thig2.7_translation-ru-narinyanievmenenko.json:10]: Please remove <j>

File: translation/ru/narinyanievmenenko/sutta/kn/thig/thig3.5_translation-ru-narinyanievmenenko.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  "thig3.5:0.2": "Три строфы ",
+  "thig3.5:0.3": "Уббири Тхери ",
+  "thig3.5:1.1": "О мать, рыдаешь ты в лесу, ",
+  "thig3.5:1.2": "Зовя дочь свою по имени Джива. ",
+  "thig3.5:1.3": "Приди скорей в себя, Уббири: ",
+  "thig3.5:1.4": "Восемьдесят тысяч дочерей погребены здесь, ",
+  "thig3.5:1.5": "И все по имени Джива —",
+  "thig3.5:1.6": "О ком из них рыдаешь ты? ",
+  "thig3.5:2.1": "Вырвана стрела эта, ",
+  "thig3.5:2.2": "Что так трудно зрима, <j>Вонзённая в моё сердце. ",
```
Comment (ccronje) [inline translation/ru/narinyanievmenenko/sutta/kn/thig/thig3.5_translation-ru-narinyanievmenenko.json:12]: <j>

File: translation/ru/narinyanievmenenko/sutta/kn/thig/thig3.5_translation-ru-narinyanievmenenko.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  "thig3.5:2.3": "Я безутешною была, <j>Скорбя по дочери, ",
```
Comment (ccronje) [inline translation/ru/narinyanievmenenko/sutta/kn/thig/thig3.5_translation-ru-narinyanievmenenko.json:13]: <j>

File: translation/ru/narinyanievmenenko/sutta/kn/thig/thig3.5_translation-ru-narinyanievmenenko.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  "thig3.5:2.4": "Но печаль моя испарилась. ",
+  "thig3.5:3.1": "Со стрелою, вырванной из сердца, ",
+  "thig3.5:3.2": "Лишённая страстей, <j>Обретшая Ниббану, ",
```
Comment (ccronje) [inline translation/ru/narinyanievmenenko/sutta/kn/thig/thig3.5_translation-ru-narinyanievmenenko.json:16]: <j>

File: translation/ru/narinyanievmenenko/sutta/kn/thig/thig4.1_translation-ru-narinyanievmenenko.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  "thig4.1:1.1": "Кассапа, Будды сын, его наследник, ",
+  "thig4.1:1.2": "Усмирённый славно —",
+  "thig4.1:1.3": "Он знает свои прошлые обители, ",
+  "thig4.1:1.4": "Он лицезрит и небеса, и бездны. ",
+  "thig4.1:2.1": "Достиг он окончания рождений, ",
+  "thig4.1:2.2": "Мудрец, обретший Высшее познанье. ",
+  "thig4.1:2.3": "Благодаря Трём обретённым знаньям, ",
+  "thig4.1:2.4": "Он именуется брахманом. ",
+  "thig4.1:3.1": "Такова и Бхадда Капилани, ",
+  "thig4.1:3.2": "Одержавшая верх над смертью, <j>Обретшая Три знанья, ",
```
Comment (ccronje) [inline translation/ru/narinyanievmenenko/sutta/kn/thig/thig4.1_translation-ru-narinyanievmenenko.json:14]: <j>

File: translation/ru/narinyanievmenenko/sutta/kn/thig/thig5.10_translation-ru-narinyanievmenenko.json [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  "thig5.10:0.2": "Пять строф ",
+  "thig5.10:0.3": "Патачара Тхери ",
+  "thig5.10:1.1": "Пашут землю люди, ",
+  "thig5.10:1.2": "В землю семена сажают. ",
+  "thig5.10:1.3": "Поддерживая детей и жён, ",
+  "thig5.10:1.4": "Богатство стяжают. ",
+  "thig5.10:2.1": "Отчего же я, добродетелью исполненная, ",
+  "thig5.10:2.2": "Наказы Учителя исполняющая, ",
+  "thig5.10:2.3": "Ниббаны не достигаю, ",
+  "thig5.10:2.4": "Хоть не праздная я, <j>Не встревоженная? ",
```
Comment (ccronje) [inline translation/ru/narinyanievmenenko/sutta/kn/thig/thig5.10_translation-ru-narinyanievmenenko.json:12]: <j>

File: translation/ru/narinyanievmenenko/sutta/kn/thig/thig5.11_translation-ru-narinyanievmenenko.json [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+  "thig5.11:3.5": "Ум пропитали безмятежностью, ",
+  "thig5.11:3.6": "Наставлениям Будды следуя. ",
+  "thig5.11:4.1": "",
+  "thig5.11:4.2": "В первой части ночи вспомнили прошлыеобители свои, ",
+  "thig5.11:4.4": "А во второй каждая обрела божественное око, ",
+  "thig5.11:4.6": "Всю тьму рассеяли в последней части ночи. ",
+  "thig5.11:5.1": "Поднявшись с мест своих, они ей в ноги кинулись: ",
+  "thig5.11:5.2": "«Исполнили мы ваше поученье, ",
+  "thig5.11:5.3": "Так будем почитать тебя, как тридцать дэвов Инду почитают! ",
+  "thig5.11:5.6": "Мы, обладательницы Трёх знаний, <j>Лишенные влечений». "
```
Comment (ccronje) [inline translation/ru/narinyanievmenenko/sutta/kn/thig/thig5.11_translation-ru-narinyanievmenenko.json:28]: <j>

File: translation/ru/narinyanievmenenko/sutta/kn/thig/thig5.12_translation-ru-narinyanievmenenko.json [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+  "thig5.12:3.3": "К ней подойдя, ",
+  "thig5.12:3.4": "О посвящении просила. ",
+  "thig5.12:4.1": "Исполненная состраданья, ",
+  "thig5.12:4.2": "Патачара посвященье мне дала; ",
+  "thig5.12:4.3": "Наставляла меня, ",
+  "thig5.12:4.4": "К высокому назначенью подвигая. ",
+  "thig5.12:5.1": "Её слова услышав, ",
+  "thig5.12:5.2": "Следовала им, ",
+  "thig5.12:5.3": "Не пустословны они были. ",
+  "thig5.12:5.4": "Я — обладательница Трёх знаний, <j>Лишённая влечений ныне. "
```
Comment (ccronje) [inline translation/ru/narinyanievmenenko/sutta/kn/thig/thig5.12_translation-ru-narinyanievmenenko.json:24]: <j>

File: translation/ru/narinyanievmenenko/sutta/kn/thig/thig5.8_translation-ru-narinyanievmenenko.json [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+  "thig5.8:0.1": "Стихи старших монахинь 5.8 ",
+  "thig5.8:0.2": "Пять строф ",
+  "thig5.8:0.3": "Сона Тхери ",
+  "thig5.8:1.1": "Десять детей я выносила в этом теле; ",
+  "thig5.8:1.3": "Бессильная, дряхлая, ",
+  "thig5.8:1.4": "К бхиккхуни приблизилась. ",
+  "thig5.8:2.1": "Она меня учила Дхамме —",
+  "thig5.8:2.2": "Совокупностям, стихиям и опорам. ",
+  "thig5.8:2.3": "Услышав Дхамму, ",
+  "thig5.8:2.4": "Я волосы свои остригла, <j>В скитания ушла. ",
```
Comment (ccronje) [inline translation/ru/narinyanievmenenko/sutta/kn/thig/thig5.8_translation-ru-narinyanievmenenko.json:11]: <j>

File: translation/ru/narinyanievmenenko/sutta/kn/thig/thig5.9_translation-ru-narinyanievmenenko.json [hunk 10]
```diff
@@ -0,0 +1,24 @@
+{
+  "thig5.9:0.1": "Стихи старших монахинь 5.9 ",
+  "thig5.9:0.2": "Пять строф ",
+  "thig5.9:0.3": "Бхадда Кундалакеса Тхери ",
+  "thig5.9:1.1": "Со спутанными волосами, <j>Что покрыты грязью, ",
```
Comment (ccronje) [inline translation/ru/narinyanievmenenko/sutta/kn/thig/thig5.9_translation-ru-narinyanievmenenko.json:5]: <j>

**Comment by neptunao** [comment]
Hi @ccronje ! I saw <j> used by bhante Sujato for line breaks (as I understood it). Do you know how can I alternatively insert a line break in Bilara?

**Pull request merged**